cmd/azure-keyvault-controller/controller/azureKeyVaultSecret.go

```diff
@@ -229,21 +229,28 @@ func (c *Controller) syncAzureKeyVault(key string) error {
 			klog.InfoS("updating with recent changes from azure key vault", "azurekeyvaultsecret", klog.KObj(akvs), "secret", klog.KRef(akvs.Namespace, akvs.Spec.Output.Secret.Name))
 			existingSecret, err := c.kubeclientset.CoreV1().Secrets(akvs.Namespace).Get(context.TODO(), akvs.Spec.Output.Secret.Name, metav1.GetOptions{})
 			if err != nil {
-				return fmt.Errorf("failed to get existing secret %s, error: %+v", akvs.Spec.Output.Secret.Name, err)
-			}
+				klog.InfoS("failed to get existing secret %s, error: %+v", akvs.Spec.Output.Secret.Name, err)
```

Comment:
Use the `klog.ErrorS` if this is an error meant for stderr. `InfoS` doesn't support formatting.
```go
klog.ErrorS(err, "failed to get existing secret", "secret", akvs.Spec.Output.Secret.Name)
```

Reply:
Updated with `klog.Infof` to enable formatting. I believe informational message is more appropriate in this case.